specification/_json_spec/query_rules.test.json

```diff
@@ -0,0 +1,32 @@
+{
+  "query_rules.test": {
```

Comment:
@kderusso This looks really useful! Are you planning to add this endpoint to the Elasticsearch specification too?

Reply:
Yes, I will add an action item to do this!